layouts/_default/list.html

```diff
@@ -49,7 +49,7 @@ <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
                                         </p>
                                         {{ if isset .Params "date" }}
                                         <p class="date-comments">
-                                            <a href="{{ .Permalink }}"><i class="far fa-calendar"></i> {{ .Date.Format .Site.Params.date_format }}</a>
```

Comment:
@GeorgeWL far is from font awesome > v5 => https://fontawesome.com/icons/calendar?style=regular